api/api.go

```diff
@@ -34,18 +34,24 @@ LICENSE
 package main
 
 import (
+	"github.com/ausocean/openfish/api/ds_client"
 	"github.com/ausocean/openfish/api/handlers"
 
+	"flag"
+	"fmt"
+
 	"github.com/gofiber/fiber/v2"
```

Comment:
Are you only using gofiber for contexts? If so, why not stay with golang.org/x/net/context which we already use?

Reply:
Or preferrably, just ["context"](https://pkg.go.dev/context).

Reply:
I'm using gofiber for its routing and its contexts